src/OSPSuite.Presentation/Presenters/Importer/IModalImporterPresenter.cs

```diff
@@ -4,15 +4,22 @@
 using OSPSuite.Infrastructure.Import.Core;
 using OSPSuite.Infrastructure.Import.Core.Mappers;
 using OSPSuite.Presentation.Views.Importer;
+using OSPSuite.Utility.Extensions;
 using System;
 using System.Collections.Generic;
+using System.Linq;
 using ImporterConfiguration = OSPSuite.Core.Import.ImporterConfiguration;
 
 namespace OSPSuite.Presentation.Presenters.Importer
 {
    public interface IModalImporterPresenter : IDisposablePresenter
    {
-      (IReadOnlyList<DataRepository> DataRepositories, ImporterConfiguration Configuration) ImportDataSets(IImporterPresenter presenter, IReadOnlyList<MetaDataCategory> metaDataCategories, IReadOnlyList<ColumnInfo> columnInfos, DataImporterSettings dataImporterSettings);
+      (IReadOnlyList<DataRepository> DataRepositories, ImporterConfiguration Configuration) ImportDataSets(
+         IImporterPresenter presenter, 
```

Comment:
Didn't we change this with a config object?

Reply:
yeap. the config object is DataImporterSettings, it was changed to also contain molweight, moleculeName and promptForConfirmation.